05/app.js

```diff
@@ -0,0 +1,46 @@
+const form = document.querySelector('form');
+const messagesEl = document.querySelector('.messages');
+
+form.setAttribute('novalidate', '');
+
+form.addEventListener('submit', function(event) {
+    messagesEl.innerHTML = '';
+
+    const formFields = Array.from(form.querySelectorAll('input, select'));
+
+    formFields.forEach(function(input) {
+        if (input.type !== 'submit') {
+            const fieldValue = getFieldValue(input);
+            const fieldName = input.name;
+            
+            if (fieldValue === '') {
```

Comment:
Pytanie czy 0 to też poprawna nazwa dla numeru budynku czy mieszkania ;) Obecnie to przechodzi ;)
Proponuję dla wprawy napisać walidację w ten sposób: https://devmentor.pl/b/walidacja-formularza-w-javascript (na pewno się przyda w następnych projektach)